manifests/mirror.pp

```diff
@@ -46,19 +46,12 @@
   $release          = $::lsbdistcodename,
   $repos            = [],
   $environment      = [],
-  $cmd_options      = {},
+  $cli_options      = {},
 ) {
   validate_string($keyserver)
   validate_array($repos)
   validate_array($environment)
-  validate_hash($cmd_options)
-
-  $default_cmd_options = {
-    '-architectures'    => '',
-    '-with-sources'     => false,
-    '-with-udebs'       => false,
-    '-force-components' => false,
-  }
+  validate_hash($cli_options)
```

Comment:
I don't want to change the module to support the Puppet 4 syntax till the functionality we want works well.